gradio/examples.py

```diff
@@ -278,10 +277,15 @@ async def predict_example(self, example_id: int) -> List[Any]:
         if len(self.outputs) == 1:
             predictions = [predictions]
         if not self._api_mode:
-            predictions = [
-                output_component.postprocess(predictions[i])
-                for i, output_component in enumerate(self.outputs)
-            ]
+            predictions_ = []
+            for i, output_component in enumerate(self.outputs):
```

Comment:
Just realized this won't work if you return a dictionary of updates. I think we should file but not fix immediately since I think it's low priority.

Reply:
Thanks for the catch here as well. Fixed now + added a test